Rakefile

```diff
@@ -103,43 +102,25 @@ bundle exec fastlane run configure_apply force:true
     end
   end
 
-  namespace :pod do
-    task :check do
-      unless podfile_locked? && lockfiles_match?
-        dependency_failed('CocoaPods')
-        Rake::Task['dependencies:pod:install'].invoke
-      end
-    end
+  desc 'Download and extract Gutenberg xcframeworks'
+  task :gutenberg_xcframeworks do
+    puts 'Setting up Gutenberg xcframeworks...'
 
-    task :install do
-      fold('install.cocoapods') do
-        pod %w[install]
-      rescue StandardError
-        puts "`pod install` failed. Will attempt to update the Gutenberg-Mobile XCFramework — a common reason for the failure — then retrying…\n\n"
-        Rake::Task['dependencies:pod:update_gutenberg'].invoke
-        pod %w[install]
-      end
-    end
+    frameworks_dir = 'WordPress/Frameworks'
+    gutenberg_tar_gz_download_path = "#{frameworks_dir}/Gutenberg.tar.gz"
 
-    task :update_gutenberg do
-      pod %w[update Gutenberg]
-    end
+    sh "mkdir -p #{frameworks_dir}"
+    sh "curl https://cdn.a8c-ci.services/gutenberg-mobile/Gutenberg-#{GUTENBERG_VERSION}.tar.gz --output #{gutenberg_tar_gz_download_path} -C -"
+    sh "tar -xf #{gutenberg_tar_gz_download_path} -C #{frameworks_dir}/ -k"
+    sh "mv -n #{frameworks_dir}/Frameworks/*.xcframework #{frameworks_dir}/"
+    sh "rm -rf #{frameworks_dir}/Frameworks #{frameworks_dir}/dummy.txt"
```

Comment:
Nitpick: FileUtils has API for mkdir, mv, and rm commands.

Reply:
Thanks for looking into it! It'd be nice if Rake has a feature to download a file (with checksum would be even nicer) and unzip a file.